viral/offline_sequence_detection.py

```
@@ -0,0 +1,1219 @@
+        start=bayesian_config.start_spatial,
+        end=bayesian_config.end_spatial,
+    )
+
+    for mouse_name in SESSIONS_KEEP.keys():
+        # for mouse_name in ["JB036"]:
+        for stage, date in SESSIONS_KEEP[mouse_name].items():
+            print(f"Processing {mouse_name} - {stage} - {date}")
+            if date:
+                try:
+                    main(mouse_name, date, bayesian_config, grosmark_config)
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)